### DISCUSSION TITLE
gitignore 위치 변경 토의 필요

### DISCUSSION THREAD

**Discussion by Sungwooo** [DeveloperAcademy-POSTECH/toGather#50] [General]
gitignore 파일이 프로젝트 밖에 있어서 package.resolved가 제외되지가 않습니다. 
그래서 병합과정속에서 충돌이 발생할 수 있고, 처음 빌드할 때마다 다시 받느라 오래걸리는 것 같습니다.
### gitignore 파일을 프로젝트 안에 넣는 것에 대해서 이야기해봐야 될거 같습니다.

**Comment by limhyoseok** [comment]
논의해보면 좋을 것 같습니다!
- gitignore 파일이 프로젝트 밖에 있어서 package.resolved가 제외되지 않는건지?
- package.resolved를 ignore 했을 때 문제가 되지는 않는지?
